src/index.js

```diff
@@ -1,23 +1,23 @@
 import babel from 'babel-core';
-import babiliPreset from 'babel-preset-babili';
+import minifyPreset from 'babel-preset-minify';
 import { SourceMapSource, RawSource } from 'webpack-sources';
 
-export default class BabiliPlugin {
-  constructor(babiliOpts = {}, options = {}) {
-    this.babiliOpts = babiliOpts;
+export default class BabelMinifyPlugin {
+  constructor(minifyOpts = {}, options = {}) {
+    this.minifyOpts = minifyOpts;
```

Comment:
Maybe instead of `preset` && `minify` just

=> `preset: [ require('babel-preset-minify'), { ...minifyOptions } ]` `{Array}`
=> `preset: { ...minifyOptions }` `{Object}`